inc/define.php

```diff
@@ -491,7 +491,7 @@
 
 $CFG_GLPI['inventory_lockable_objects'] = ['Computer_Item',  'Item_SoftwareLicense',
     'Item_SoftwareVersion', 'Item_Disk', 'ComputerVirtualMachine',
-    'NetworkPort', 'NetworkName', 'IPAddress'
+    'NetworkPort', 'NetworkName', 'IPAddress', 'Item_OperatingSystem'
```

Comment:
Sounds like several other possible itemtypes are missing here aswell, lika all item/devices, computer antivirus, and so on.

Reply:
agree =)